### PR TITLE
update legacy button component

### PR DESCRIPTION
**Pull request by gwdawson**
removes all legacy forms and replaces them with updated components

### CODE REVIEW

**Review by gabor — APPROVED** [review]
LGTM 👍

File: src/components/FieldEditor.tsx [hunk 1]
```diff
@@ -94,16 +94,14 @@ export const FieldEditor = ({ value = [], onChange, limit, onComplete }: Props)
...
 
           {(!limit || value.length < limit) && (
-            <a className="gf-form-label" onClick={addField(index, { language: field.language ?? 'jsonpath' })}>
-              <Icon name="plus" />
-            </a>
+            <Button
+              variant="secondary"
+              onClick={addField(index, { language: field.language ?? 'jsonpath' })}
+              icon="plus"
+            />
```
Comment (zoltanbedi) [inline src/components/FieldEditor.tsx:101]:
```suggestion
            <Button
              variant="secondary"
              onClick={addField(index, { language: field.language ?? 'jsonpath' })}
              title="plus"
              icon="plus"
            />
```

File: src/components/FieldEditor.tsx [hunk 2]
```diff
@@ -94,16 +94,14 @@ export const FieldEditor = ({ value = [], onChange, limit, onComplete }: Props)
...
           )}
 
-          {value.length > 1 ? (
-            <a className="gf-form-label" onClick={removeField(index)}>
-              <Icon name="minus" />
-            </a>
-          ) : null}
+          {value.length > 1 ? <Button variant="secondary" onClick={removeField(index)} icon="minus" /> : null}
```
Comment (zoltanbedi) [inline src/components/FieldEditor.tsx:104]:
```suggestion
          {value.length > 1 ? <Button variant="secondary" onClick={removeField(index)} icon="minus" title="minus" /> : null}
```

**Review by zoltanbedi — COMMENTED** [review]
Added suggestion to pass a11y check

**Pull request merged**